poetry/installation/executor.py

```diff
@@ -54,6 +56,7 @@ def __init__(
         config: "Config",
         io: "IO",
         parallel: bool = None,
+        max_workers: Optional[int] = None,
```

Comment:
I don't see a need to make this configurable here -- unless you'd like to move the config of this value outside the class. I'd argue against any additional churn, however.

Reply:
At first I thought to add it in the same as `parallel`, which is explicitly added but also unused right now. But for now I've removed it.